### ISSUE TITLE
[ULTMOS] Update from v0.11.0 to v0.11.1 fails with `'Namespace' object has no attribute 'targetBranch'`

### ISSUE THREAD

**Issue by adil192** [Coopydood/ultimate-macOS-KVM#77]
### Describe the Issue

When I try to update with `main.py`, I get this error:

```console
   ⧖  UPDATING... 
   Do not terminate this script

         Current Version
             v0.11.0 

               ⋁

         Target Version
             v0.11.1 

   The update you requested is being downloaded and installed.
   Do NOT terminate this script or close the window.


Traceback (most recent call last):
  File "/home/ahann/Documents/Sources/ultimate-macOS-KVM/./scripts/repo-update.py", line 477, in <module>
    updateBrains()
  File "/home/ahann/Documents/Sources/ultimate-macOS-KVM/./scripts/repo-update.py", line 236, in updateBrains
    elif args.targetBranch is not None:
AttributeError: 'Namespace' object has no attribute 'targetBranch'
```

### Operating System

Pop OS

### Kernel

Linux pop-os 6.6.10-76060610-generic #202401051437~1709085277~22.04~31d73d8 SMP PREEMPT_DYNAMIC Wed F x86_64 x86_64 x86_64 GNU/Linux

### Processor

Intel i7-11800H

### RAM (in GB)

16

### Version

v0.11.0

### Branch

<s>main</s> I actually checked out v0.11.0 because I was having issues with the main branch

### Additional Information

Running `git checkout main` doesn't fix the built-in updater, but pulling the latest changes of the main branch seems to work.

**Comment by Coopydood** [comment]
I'll look into this, thank you

**Comment by Coopydood** [comment]
Fix implemented in v0.11.2, thank you
